python_modules/dagster/dagster/core/materializable.py

```diff
@@ -25,20 +25,26 @@ def materialize_runtime_value(self, _config_spec, _runtime_value):
 
 
 class MaterializeableBuiltinScalarConfigSchema(ConfigurableSelectorFromDict):
-    def __init__(self):
+    def __init__(self, name):
```

Comment:
All these change are a bit problematic and cropped up when GraphQL queried the type system. We need to have the graphql schema to reflect the new reality of the type system.

Reply:
Addressing the issue in this branch?